extensions/transform/org.eclipse.smarthome.transform.xslt/README.md

```diff
@@ -103,6 +103,21 @@ Fri, 18 Dec 2009 9:38 am PST
 450
 ```
 
+## Usage as a Profile
+
+The functionality of this `TransformationService` can be used in a `Profile` on an `ItemChannelLink` too.
+To do so, it can be configured in the `.items` file as follows:
+
+```java
+String <itemName> { channel="<channelUID>"[profile="transform:XSLT", function="<xsltExpression>", sourceFormat="<valueFormat>"]}
+```
+
+The XSLT expression to be executed has to be set in the `function` parameter.
```

Comment:
in the config description the function parameter is supposed to hold a filename. Here it should be the XSLT expression...

edit: the XsltTransformationService expects it to be the file name.